test/cxx/netcdf_test_fixture.cc

```
@@ -0,0 +1,37 @@
+    test_int_var_data = {1, 2, 3, 4, 5};
+    test_int64_var_data = {11111, 22222, 33333, 44444, 55555};
+    test_real_var_data = {1.1f, 2.2f, 3.3f, 4.4f, 5.5f};
+    test_string_var_data = {"one", "two", "three", "four", "five"};
+
+    test_int_att_name = "int_var";
+    test_string_att_name = "string_att";
+    test_int_att_data = 2;
+    test_string_att_data = "string";
+}
+
+void NetCDFTestFixture::TearDown() {
```

cppcheck (2.10-2)

[style] The function 'TearDown' is never used.

(unusedFunction)